contrib/editors/kedit/neon.kld

```diff
@@ -87,24 +88,18 @@
  Boolean        alternate 2
  Bytes          alternate 2
  Dictionary     alternate 6
+ HEXBYTES       alternate 2
  Number         alternate 2
  String         alternate 2
 * NIL is actually a unique value literal, used only with pointer operations.
  NIL            alternate 2
- FIRST          alternate 2
- INDEX          alternate 2
- LAST           alternate 2
- OF             alternate 2
```

Comment:
It got moved to an actual language keyword, rather than a post-processed operator...